src/benchmark/micro_benchmark_utils.cpp

```diff
@@ -12,4 +17,23 @@ void micro_benchmark_clear_cache() {
   }
 }
 
+void micro_benchmark_clear_disk_cache() {
+  // TODO(phoenix): better documentation of which caches we are clearing
+  sync();
+  std::ofstream ofs("/proc/sys/vm/drop_caches");
+  ofs << "3" << std::endl;
+}
+
+/**
+ * Generates random indexes between 0 and number.
+*/
+std::vector<int> generate_random_indexes(int number) {
```

Comment:
So what would happen if I pass a negative number? Since you accept an `int` that would adhere to your interface. :)